chef_master/source/resource_systemd_unit.rst

```diff
@@ -126,7 +126,7 @@ This resource has the following properties:
 ``content``
    **Ruby Type:** String, Hash
 
-   A string or hash that contains a systemd `unit file <https://www.freedesktop.org/software/systemd/man/systemd.unit.html>`_ definition that describes the properties of systemd-managed entities, such as services, sockets, devices, and so on.
+   A string or hash that contains a systemd `unit file <https://www.freedesktop.org/software/systemd/man/systemd.unit.html>`_ definition that describes the properties of systemd-managed entities, such as services, sockets, devices, and so on. Repeatable options can be implemented with an array.
```

Comment:
You might want to start that last statement with: In Chef 14.4... so that users know what version they need before they can use arrays.

Reply:
OK.